pkg/antctl/raw/multicluster/deploy/member_cluster.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package deploy
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+type memberClusterOptions struct {
+	namespace     string
+	antreaVersion string
+	filename      string
+}
+
+var memberClusterOpts *memberClusterOptions
+
+var memberClusterExamples = strings.Trim(`
+Create the member cluster CRDs and deploy the "antrea-mc-controller" Deployment in Namespace antrea-mcs
+$ antctl mc deploy member-cluster --antrea-version v1.6.0 -n antrea-mcs
```

Comment:
Better to use the default Namespace `kube-system` in the sample for member cluster.

Reply:
done